### PR TITLE
Image

### PR DESCRIPTION
**Pull request by gdub01**
Takes width, height, src and alt. Width and height are currently mandatory but wasn't sure if that should always be true. I think it makes sense for the db to know the _stored_ image sizes so I'm imagining when we request the image block, we get the original image sizes back. The client can then determine how/if it wants to transform the image... either by modifying the source url or by changing some properties it adds to the next/image component. But I'm thinking we'll deal with some of those things later.

### CODE REVIEW

**Review by storyworks — APPROVED** [review]
Just need to merge in main before you merge - there's some updates to BlockRenderer

**Comment by tataihono** [comment]
I think the backend is going to fail. Did you manage to test this before approval @storyworks?

File: apps/journeys/src/components/blocks/Image/Image.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const Demo = {
+  ...journeysConfig,
+  component: Image,
+  title: 'Journeys/Blocks/Image'
+}
+
+const DefaultTemplate: Story<TreeBlock<ImageBlock>> = ({
+  ...props
+  // eslint-disable-next-line jsx-a11y/alt-text
```
Comment (tataihono) [inline apps/journeys/src/components/blocks/Image/Image.stories.tsx:15]: I don't think we should be adding in eslint disables here.

**Review by tataihono — DISMISSED** [review]

File: apps/journeys/src/components/blocks/Image/Image.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        borderRadius: (theme) => theme.spacing(3),
+        overflow: 'hidden',
+        mb: 4,
+        ...sx,
+        '> div': {
+          display: 'block !important'
+        }
+      }}
+    >
+      <NextImage src={src} alt={alt} height={height} width={width} />
```
Comment (storyworks) [inline apps/journeys/src/components/blocks/Image/Image.tsx:31]: Add `objectFit='cover'` ?

**Review by storyworks — CHANGES_REQUESTED** [review]
There's also the issue of the conductor container cutting off the shadows on Cards, but I'm not sure if you want to fix that in another PR. https://github.com/JesusFilm/core/pull/36#discussion_r719867474

File: apps/journeys/src/components/blocks/Image/Image.tsx (hunk 2 above)
Comment (storyworks) [reply apps/journeys/src/components/blocks/Image/Image.tsx:31]: Actually, with that - maybe use a picture that makes it more obvious if it's been squashed in some way? It's really easy to miss with this one..

**Review by storyworks — APPROVED** [review]

**Pull request merged**